beat/models/seismic.py

```diff
@@ -265,16 +291,31 @@ def save_covs(wmap, cov_mat='pred_v'):
             """
             Save covariance matrixes of given attribute
             """
-            covs = {
-                utility.list2string(dataset.nslc_id):
-                    getattr(dataset.covariance, cov_mat)
-                for dataset in wmap.datasets}
+            if wmap.config.timedomain_include:
```

Comment:
again here only if else ... all throughout ...
in general as a programming style if if in this way is not a good style! you would only use:
if
elif (optional)
else
if there is nothing here think about at leaast a logger.debug statement to put here or is there a case that may justify raising an error? ...(as a general remark)